### ISSUE TITLE
setup sideloading  with existing OS versions fails

### ISSUE THREAD

**Issue by roelofz** [elementary/installer#869]
### What Happened?

Hi, 

If you don´t select a custom install setup will fail (at least in my case.
If you do select a custom installation ánd you setup a partition, it failed the first time (after closing Gparted, the other screen was refreshed and disappeared.
The second time (reboot) it failed again, after selecting the EFI and the root partition (what misses?), no info was given why.
The third time I selected the EFI, the root partition and the (prepared) swap partition. 

Now this is a user friendly OS, and this should be in the install wizard too.
So I was warned in (very and too negative) youtube video's, and really wanted this experience.
It should be more polished in order to get more users quickly setup.

Bottom line, happy to run it finally bare metal, and very enthusiastic, now coming from fedora (and for work Windows).

### Steps to Reproduce

Descibed above

### Expected Behavior

A smooth setup, even in side loading.

### OS Version

8.x (Circe)

### Session Type

Secure Session (Wayland)

### Software Version

Latest release (I have run all updates)
